src/components/gme/global/footer/news-letter.tsx

```diff
@@ -21,41 +21,41 @@ const Newsletter: React.FC<SubscriptionBannerProps> = ({
     buttonText = 'Subscribe Now',
     buttonBg = 'bg-black',
     buttonTextColor = 'text-white',
-    bannerBg = 'bg-red-700',
+    bannerBg = 'bg-[#ce181e]',
```

Comment:
```suggestion
    bannerBg = 'bg-primary',
```
Please replace bg-[#ce181e] with bg-primary to keep it consistent with the design system.